<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Quta.D300M.Card.Tools</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnwrite">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>写卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnread">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>读卡</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="writecode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="readcode">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>510</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="msgline">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tbview">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>570</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="getgroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>获取公司</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="buildview">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Maps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnpre">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>510</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnnext">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>写块数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始块</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读/写区块数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读/写开始区块</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="qreadsector">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="qwritesector">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="qreadtrack">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="qwritetrack">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="qkeys">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>范例：ffffffffffff  只允许数字和a-f之间小写字母，长度12位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnactive">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置主动工作模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btpassive">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置被动工作模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnconfigwrite">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存写卡配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnconfigread">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存读卡配置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N*4+3和0块不可用(N=0-15)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>正序</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>反序</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="readdefault">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读物理卡号</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="readcustor">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读配置卡号</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>242</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清卡</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
